Project.toml

```diff
@@ -26,7 +26,7 @@ Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 
 
 [compat]
-DataFrames = "1.1.0"
+DataFrames = "0.22,1.0.0,1.1.0"
```

Comment:
I think we can just include major and minor version, leaving off the patch version. So, perhaps replace with `"0.22,1.0,1.1"`